scripts/broker.csx

```diff
@@ -0,0 +1,111 @@
+#r "../src/packages/NServiceBus.6.0.0/lib/net452/NServiceBus.Core.dll"
+#r "../src/packages/NServiceBus.AcceptanceTesting.6.0.0/lib/net452/NServiceBus.AcceptanceTesting.dll"
```

Comment:
Required for [`RunSettings`](https://github.com/Particular/NServiceBus.RabbitMQ/pull/275/files#diff-4aacd7a8808fd077b2ab57dc39c949b1R34).